src/groups/mwc/mwcio/mwcio_testchannel.h

```diff
@@ -248,6 +248,10 @@ class TestChannel : public Channel {
     /// Pops a close-call from those written to the channel (FIFO ordering).
     CloseCall popCloseCall();
 
+    /// Lock mutex and return `true` if d_closeCalls collection is empty,
+    /// `false` otherwise.
+    bool closeCallsEmpty();
```

Comment:
Might qualify as `const` and make mutex `mutable`

Reply:
Fixed as proposed.